### ISSUE TITLE
[Bug] Auto remove _ from highlight candidate

### ISSUE THREAD

**Issue by tmpm697** [searchmarkers/mark-my-search#129]
Steps to produce:
```
1. chromium -> https://registry.terraform.io/providers/hashicorp/template/latest/docs/data-sources/cloudinit_config
2. select `template_file`, then right mouse click to select `Highlight Selection`
3. observe the search candidate appear in mark-my-search bar is `templatefile`
```
here we expect the search candidate is `template_file` not `templatefile`

**Comment by ator-dev** [comment]
Thanks, turns out I was removing too many unicode character types, including connecting punctuation :sweat_smile: 

I took the opportunity to improve tabbing support in the keywords bar.

**Comment by ator-dev** [comment]
Now available as 1.14.1 on Chrome Web Store.
